carbon_bomb.py

```diff
@@ -1117,6 +1156,18 @@ def create_carbon_bombs_table():
     df_carbon_bombs["Suppliers_source_chatGPT"] = ""
     df_carbon_bombs["Insurers_source_chatGPT"] = ""
     df_carbon_bombs["Subcontractors_source_chatGPT"] = ""
+
+    # Post process dataset to sort columns with aggregated values
+    agg_columns = [
+        "GEM_id_source_GEM",
+        "GEM_url_source_GEM",
+        "Operators_source_GEM",
+        "Parent_company_source_GEM",
+        "Multiple_unit_concerned_source_GEM"
+    ]
+    for c in agg_columns:
+        df_carbon_bombs[c] = sort_agg_values_in_column(df_carbon_bombs[c])
```

Comment:
When I see how you used the function I think there is a better way to apply your transformation by directly using  df_carbon_bombs[c] = df_carbon_bombs[c].apply(sort_values_if_not_null(). Should be test but It's more simple if when modifying the columns we see the function .apply. We can discuss this point I also might be wrong.